plansys2_popf_plan_solver/src/plansys2_popf_plan_solver/popf_plan_solver.cpp

```diff
@@ -36,10 +37,13 @@ POPFPlanSolver::getPlan(
   const std::string & node_namespace)
 {
   if (node_namespace != "") {
-    //  This doesn't work as cxx flags must apper at end of link options, and I didn't
-    //  find a way
-    // std::experimental::filesystem::create_directories("/tmp/" + node_namespace);
-    mkdir(("/tmp/" + node_namespace).c_str(), ACCESSPERMS);
+    std::filesystem::path tp = std::filesystem::temp_directory_path();
+    for (auto p : std::filesystem::path(node_namespace) ) {
+      if (p != std::filesystem::current_path().root_directory()) {
+        tp /= p;
+      }
```

Comment:
Only append the "namespace" part, if it do not corresponds to the root dirretory ("/").